default.nix

```diff
@@ -15,11 +15,15 @@ let
       overrides = self: super:
         {
           prettyprinter = self.prettyprinter_1_7_0; # at least 1.7
-          hnix = appendPatches super.hnix [
-            ./hnix-fix-qq.patch
-            ./hnix-generic1-nkeyname.patch
-            ./hnix-lexeme-sourcepos.patch
-          ];
+          hnix = overrideSrc super.hnix {
+            src = pkgs.fetchFromGitHub {
+              owner = "expipiplus1";
+              repo = "hnix";
+              rev =
+                "9bbb9f8d4a22f88dce5004996c57b9a48eacbf4f"; # update-nix-fetchgit
+              sha256 = "1xwcf3mih7zrc5dfc75g2wssl270aifsz4i00maqrw4w6bwaff0k";
```

Comment:
Was so satisfying to just write `rev = _; # update-nix-fetchgit;` `sha256 = _;` and then run in vim `:!update-nix-fetchgit` and see it all appear :)

Reply:
Love it! Next level time saver.